### ISSUE TITLE
Use zero-code JS instrumentation for Payment service.

### ISSUE THREAD

**Issue by puckpuck** [open-telemetry/opentelemetry-demo#1605]
# Feature Request

Currently the payment service has an `opentelemetry.js` code file which is required as part of the node app startup. This file is the standard boilerplate code except for the config on a single auto-instrumentation library. 

All of that same functionality can be achieved by leveraging environment variables and the zero-code approach, which requires a file part of the `opentelemetry/auto-instrumentations-node` package. We can even showcase how to do this using environment variables only with `NODE_OPTIONS`. Here are the docs on how to get zero-code instrumentation for Node https://opentelemetry.io/docs/zero-code/js/#configuring-the-module

**Comment by tlee768** [comment]
Is this still desired?  If so I can take a stab

**Comment by puckpuck** [comment]
Yes this still hasn't been done, and it would be great to have a JS service that is properly auto-instrumented. Assigned this to @tlee768

**Comment by tlee768** [comment]
@puckpuck Rdy for review pls:

https://github.com/open-telemetry/opentelemetry-demo/pull/3486
